great_expectations/expectations/expectation.py

```diff
@@ -1367,6 +1367,188 @@ def _validate(
+
+    def validate_configuration(self, configuration: Optional[ExpectationConfiguration]):
+        if not super().validate_configuration(configuration):
+            return False
+        try:
+            assert (
+                "column_list" in configuration.kwargs
+            ), "'column_list' parameter is required for multicolumn map expectations"
+        except AssertionError as e:
+            raise InvalidExpectationConfigurationError(str(e))
+        return True
+
+    def get_validation_dependencies(
+        self,
+        configuration: Optional[ExpectationConfiguration] = None,
+        execution_engine: Optional[ExecutionEngine] = None,
+        runtime_configuration: Optional[dict] = None,
+    ):
+        dependencies = super().get_validation_dependencies(
+            configuration, execution_engine, runtime_configuration
+        )
+        assert isinstance(
+            self.map_metric, str
+        ), "MulticolumnMapExpectation must override get_validation_dependencies or declare exactly one map_metric"
+        assert (
+            self.metric_dependencies == tuple()
+        ), "MulticolumnMapExpectation must be configured using map_metric, and cannot have metric_dependencies declared."
+        # convenient name for updates
+
+        result_format_str = dependencies["result_format"].get("result_format")
+
+        metric_dependencies = dependencies["metrics"]
+
+        metric_kwargs = get_metric_kwargs(
+            metric_name=self.map_metric + ".unexpected_count",
+            configuration=configuration,
+            runtime_configuration=runtime_configuration,
+        )
+        metric_dependencies[
+            self.map_metric + ".unexpected_count"
+        ] = MetricConfiguration(
+            self.map_metric + ".unexpected_count",
+            metric_domain_kwargs=metric_kwargs["metric_domain_kwargs"],
+            metric_value_kwargs=metric_kwargs["metric_value_kwargs"],
+        )
+
+        metric_kwargs = get_metric_kwargs(
+            metric_name="table.row_count",
+            configuration=configuration,
+            runtime_configuration=runtime_configuration,
+        )
+        metric_dependencies["table.row_count"] = MetricConfiguration(
+            metric_name="table.row_count",
+            metric_domain_kwargs=metric_kwargs["metric_domain_kwargs"],
+            metric_value_kwargs=metric_kwargs["metric_value_kwargs"],
+        )
+
+        metric_kwargs = get_metric_kwargs(
+            self.map_metric + ".filtered_row_count",
+            configuration=configuration,
+            runtime_configuration=runtime_configuration,
+        )
+        metric_dependencies[
+            self.map_metric + ".filtered_row_count"
+        ] = MetricConfiguration(
+            metric_name=self.map_metric + ".filtered_row_count",
+            metric_domain_kwargs=metric_kwargs["metric_domain_kwargs"],
+            metric_value_kwargs=metric_kwargs["metric_value_kwargs"],
+        )
```

Comment:
I think this approach is *much* more readable. Like, *so* much more readable than the way the `ColumnMapExpectation` handles this that I think we should add "filtered_row_count" to all `MapMetricProvider` children. If you agree, let's add it to the JIRA regarding streamlining the code that handles the metric registration process.

Reply:
@jcampbell Agreed.  This is now in JIRA: `https://superconductive.atlassian.net/browse/GE-439`